### PR TITLE
更新方針1

### PR DESCRIPTION
**Pull request by hanwu85**

### CODE REVIEW

**Review by richar983 — CHANGES_REQUESTED** [review]
就是測試一下xDD

結果居然還刪不掉xDDDD

**Comment by richar983** [comment]
原文：

1h1+++) [ADDITION] Competitors may request to compete at a time different to the scheduled time for the round in a separate group. The WCA Delegate should carefully consider the fairness and practicality of the situation before approving any such requests, and must ensure that such attempts follow [Regulation 9l](regulations:regulation:9l) (i.e. they must start after all competitors have finished their attempts from any previous round of the same event, and finish before any competitor has started an attempt for any following round of the same event). Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition. Exception: such accommodations are not permitted for 3x3x3 Fewest Moves, which specifies its own limited accommodations in [Guideline E2b++](guidelines:guideline:E2b++).

**Comment by richar983** [comment]
WCA #1074

原文：

`1h1+++) [ADDITION] Competitors may request to compete at a time different to the scheduled time for the round in a separate group. The WCA Delegate should carefully consider the fairness and practicality of the situation before approving any such requests, and must ensure that such attempts follow [Regulation 9l](regulations:regulation:9l) (i.e. they must start after all competitors have finished their attempts from any previous round of the same event, and finish before any competitor has started an attempt for any following round of the same event). Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition. Exception: such accommodations are not permitted for 3x3x3 Fewest Moves, which specifies its own limited accommodations in [Guideline E2b++](guidelines:guideline:E2b++).`

File: wca-guidelines.md [hunk 1]
```diff
@@ -38,7 +38,7 @@
 - 1h++) [建議] 每個比賽項目的決賽以及每一輪的最少步數解都應該要讓所有參賽者使用相同的轉亂步驟（也就是只能有一組）。
 - 1h1+) [釐清] 在為了使比賽能順利進行下，才能讓轉亂員或裁判為與自己同一組的參賽者做轉亂或當裁判。
 - 1h1++) [釐清] 對於最少步數解，裁判可以在他完成該還原之前判同組參賽者（也就是裁判自己同時也在比賽）。
-
+- 1h+++) [附加] 參賽者可以要求在同一輪中，加入不同時間的參賽組別。在WCA代表的同意下，可以允許參賽者參賽。WCA代表應該謹慎考慮該要求的比賽公平性和實行的可行性，且必須要確保這個改變有符合[規則 9l](regulations:regulation:9l)(也就是說該參賽者必須要在同一項目的前一輪全部結束以及下一輪還未開始以前結束還原)。有該需求的參賽者應該在比賽前聯絡主辦團隊和WCA代表。例外：最少步數解不能使用這個安排因為 [指導方針 E2b++](guidelines:guideline:E2b++)中有獨自的安排方式。
```
Comment (richar983) [inline wca-guidelines.md]: 原文：
`
1h1+++) [ADDITION] Competitors may request to compete at a time different to the scheduled time for the round in a separate group. The WCA Delegate should carefully consider the fairness and practicality of the situation before approving any such requests, and must ensure that such attempts follow [Regulation 9l](regulations:regulation:9l) (i.e. they must start after all competitors have finished their attempts from any previous round of the same event, and finish before any competitor has started an attempt for any following round of the same event). Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition. Exception: such accommodations are not permitted for 3x3x3 Fewest Moves, which specifies its own limited accommodations in [Guideline E2b++](guidelines:guideline:E2b++).
`

```suggestion
- 1h+++) [附加] 在WCA代表的同意下，參賽者可以要求在同一輪比賽中參加不同的組別。WCA代表應該謹慎考慮該要求對比賽的公平性與實行的可行性，且必須要確保這個更動有符合[規則 9l](regulations:regulation:9l)(也就是該參賽者必須要在同一項目的前一輪全部結束且下一輪還未開始以前結束還原)。有此需求的參賽者應該要在比賽前聯絡主辦團隊和WCA代表。例外：最少步數解不能使用這個安排因為 [指導方針 E2b++](guidelines:guideline:E2b++)中有獨自的安排方式。
```

File: wca-guidelines.md [hunk 2]
```diff
@@ -36,6 +36,7 @@
 - 1c++) [釐清] WCA代表們即使沒有被指派為該比賽的WCA代表也可以參加該場比賽。這些人並非「WCA代表」，除非他們被指定的WCA代表指派負責該比賽的相關事宜。(參見[指導方針 1c+](guidelines:guideline:1c+))
 - 1h+) [建議] 同一組別的參賽者，應該要使用同樣的轉亂步驟。不同組別應該要使用不同的轉亂步驟。
 - 1h++) [建議] 每個比賽項目的決賽以及每一輪的最少步數解都應該要讓所有參賽者使用相同的轉亂步驟（也就是只能有一組）。
+- 1h+++) [附加] 參賽者可以要求在同一輪中，加入不同時間的參賽組別。在WCA代表的同意下，可以允許參賽者參賽。WCA代表應該謹慎考慮該要求的比賽公平性和實行的可行性，且必須要確保這個改變有符合[規則 9l](regulations:regulation:9l)(也就是說該參賽者必須要在同一項目的前一輪全部結束以及下一輪還未開始以前結束還原)。有該需求的參賽者應該在比賽前聯絡主辦團隊和WCA代表。例外：最少步數解不能使用這個安排因為 [指導方針 E2b++](guidelines:guideline:E2b++)中有獨自的安排方式。
```
Comment (richar983) [inline wca-guidelines.md]:
```suggestion
- 1h+++) [附加] 在WCA代表的同意下，參賽者可以要求在同一輪比賽中參加不同的組別。WCA代表應該謹慎考慮該要求對比賽的公平性與實施的可行性，且必須要確保這個更動有符合[規則 9l](regulations:regulation:9l)(也就是該參賽者必須要在同一項目的前一輪全部結束且下一輪還未開始以前結束還原)。有此需求的參賽者應該要在比賽前聯絡主辦團隊和WCA代表。例外：最少步數解不能使用這個安排因為 [指導方針 E2b++](guidelines:guideline:E2b++)中有獨自的安排方式。
```

其中有幾個小地方可能值得探討看看…

`參加不同的組別` 或 `參加不同時間的組別`

`實行的可能性` 或 `實施的可能性`

**Review by richar983 — CHANGES_REQUESTED** [review]

File: wca-guidelines.md (hunk 2 above)
Comment (hanwu85) [reply wca-guidelines.md]: `實行的可能性` 或 `實施的可能性` 這兩個我都OK
`參加不同的組別` 或 `參加不同時間的組別`  雖然不同組別通常都是不同時間，但我覺得還是要刻意強調一下是不同時間，畢竟原文也有寫"不同時間"

Comment (hanwu85) [reply wca-guidelines.md]: 我的建議
```suggestion
- 1h+++) [附加] 在WCA代表的同意下，參賽者可以要求在同一輪比賽中參加不同時間的組別。WCA代表應該謹慎考慮該要求對比賽的公平性與實施的可行性，且必須要確保這個更動有符合[規則 9l](regulations:regulation:9l)(也就是該參賽者必須要在同一項目的前一輪全部結束且下一輪還未開始以前結束還原)。有此需求的參賽者應該要在比賽前聯絡主辦團隊和WCA代表。例外：最少步數解不能使用這個安排因為 [指導方針 E2b++](guidelines:guideline:E2b++)中有獨自的安排方式。
```

Comment (richar983) [reply wca-guidelines.md]: 看起來是挺不錯的！

**Review by richar983 — APPROVED** [review]

**Pull request merged**